tests/components/vera/common.py

```diff
@@ -0,0 +1,55 @@
+"""Common code for tests."""
+
+from typing import Callable, NamedTuple, Tuple
+
+from mock import MagicMock, patch
+from pyvera import VeraController, VeraDevice, VeraScene
+
+from homeassistant.components.vera import CONF_CONTROLLER, DOMAIN
+from homeassistant.const import CONF_UNIT_SYSTEM, CONF_UNIT_SYSTEM_METRIC
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+ComponentData = NamedTuple("ComponentData", (("controller", VeraController),))
+
+
+async def async_configure_component(
+    hass: HomeAssistant,
+    devices: Tuple[VeraDevice] = (),
+    scenes: Tuple[VeraScene] = (),
+    setup_callback: Callable[[VeraController], None] = None,
+) -> ComponentData:
+    """Configure the component with specific mock data."""
+    controller_url = "http://127.0.0.1:123"
+
+    hass_config = {
+        "homeassistant": {CONF_UNIT_SYSTEM: CONF_UNIT_SYSTEM_METRIC},
```

Comment:
Why do we need to set unit system in the config? The `hass` test instance sets the unit system on `hass.config` to metric by default.